.template/pubspec.yaml

```diff
@@ -33,6 +33,7 @@ dependencies:
   intl: ^0.17.0
   json_annotation: ^4.5.0
   package_info_plus: ^1.4.2
+  permission_handler: ^8.3.0
```

Comment:
Do we need to update pubspec.lock?

Reply:
@luongvo I guess no need, as it will be generated after getting pubspec 🙏

Reply:
@doannimble I know it automatically generates it for us. However, we're keeping these `.lock` files in Git so I don't want everyone messy with it in Git Stage Changes in later features or branches. I think you got the point, right? Same as `Pod.lock` 🤓

Reply:
@luongvo Okay, got your point now. I forgot that we are putting these files in Git too. Let me commit them